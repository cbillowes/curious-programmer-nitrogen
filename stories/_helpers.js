import React from 'react'
import Layout from '../src/components/layout'
import { SmallSpacer } from '../src/components/spacer'

const Page = ({ className, children }) => {
  return (
    <Layout
      className={className}
    >
      <SmallSpacer />
      {children}
    </Layout>
  )
}

export default Page