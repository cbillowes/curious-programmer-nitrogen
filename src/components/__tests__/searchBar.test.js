import React from 'react'
import SearchBar from '../searchBar'
import { getSnapshot } from './_helpers'

describe(`Search bar`, () => {
  it(`should render`, () => {
    const bar = (
      <SearchBar />
    )
    const tree = getSnapshot(bar)
    expect(tree).toMatchSnapshot()
  })
})
