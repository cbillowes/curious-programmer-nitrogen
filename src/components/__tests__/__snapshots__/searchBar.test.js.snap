// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search bar should render 1`] = `
Array [
  <button
    className="search-button"
  >
    <svg
      aria-hidden="true"
      className="svg-inline--fa fa-search fa-w-16 "
      data-icon="search"
      data-prefix="fas"
      focusable="false"
      role="img"
      style={Object {}}
      viewBox="0 0 512 512"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M505 442.7L405.3 343c-4.5-4.5-10.6-7-17-7H372c27.6-35.3 44-79.7 44-128C416 93.1 322.9 0 208 0S0 93.1 0 208s93.1 208 208 208c48.3 0 92.7-16.4 128-44v16.3c0 6.4 2.5 12.5 7 17l99.7 99.7c9.4 9.4 24.6 9.4 33.9 0l28.3-28.3c9.4-9.4 9.4-24.6.1-34zM208 336c-70.7 0-128-57.2-128-128 0-70.7 57.2-128 128-128 70.7 0 128 57.2 128 128 0 70.7-57.2 128-128 128z"
        fill="currentColor"
        style={Object {}}
      />
    </svg>
  </button>,
  <div
    className="search closed"
  >
    <input
      className="search-box"
      onKeyDown={[Function]}
      onKeyPress={[Function]}
      onKeyUp={[Function]}
      placeholder="Looking for something?"
    />
    <div
      className="spacer small"
    />
    <div
      className="empty-results"
    >
      <h1>
        Sorry.
      </h1>
      <p>
        So one of two things could have happend here.
        <br />
        Either the indexing has stopped working or I haven't written about it.
        <br />
        I am sorry that I canâ€™t tell you which one it is.
      </p>
    </div>
    <ul
      className="results"
    />
  </div>,
]
`;
